scipy/stats/setup.py

```diff
@@ -27,9 +32,17 @@ def configuration(parent_package='',top_path=None):
     config.add_extension('mvn',
         sources=['mvn.pyf','mvndst.f'],
     )
-        
-    # What do I add here to build Rectangle?
-    
+
+    # Build cython rectangle demo
+    config.add_data_files('rect.pxd')
+    ext = config.add_extension(
+        'rect',
+        sources=['rect.cxx'],
+    )
+    # This is how you get C++11 support:
+    ext._pre_build_hook = pre_build_hook
```

Comment:
This is how the `pre_build_hook` is hooked up to this C++ extension

Reply:
OK.

Reply:
Nope, not needed for this